### PR TITLE
Support for company fields in tracking code pageview event

### PR DESCRIPTION
**Pull request by kuzmany**
https://github.com/mautic/mautic/pull/5611

### CODE REVIEW

File: en/contacts/contact_monitoring.md [hunk 1]
```diff
@@ -62,10 +62,12 @@ You can attach custom parameters or overwrite the automatically generated parame
 ```
 This code will send all the automatic data to Mautic and adds also email and firstname. The values of those fields must be generated by your system. 
 
-From Mautic 2.13 tracking code support also company fields. Mautic assign company to your tracking contact based on own rules for find contact's company from  paramteres:
- - company or companyname
- - If parameters above weren't set and email or company email were set up, then try find from email or companyemail address based on domain name (company name from **david@mautic.com** is **mautic**)
- - If logice above weren't set and company website were set up, then try find from companywebsite  (company name from **www.mautic.com** is company name **mautic**)
+From Mautic 2.13 tracking code support also company fields. Mautic assign company to your tracking contact based on company name. Then you have to add **company ** or **companyname** parameter to tracking code along with other companies fields (companyemail, companyaddress1)
```
Comment (escopecz) [inline en/contacts/contact_monitoring.md]: Please remove the space from `**company **`

File: en/contacts/contact_monitoring.md [hunk 2]
```diff
@@ -62,10 +62,12 @@ You can attach custom parameters or overwrite the automatically generated parame
...
+
+```
+    mt('send', 'pageview', {email: 'my@email.com', firstname: 'John', company: 'Mautic', companywebsite: 'mautic@mautic.com'});
```
Comment (escopecz) [inline en/contacts/contact_monitoring.md]: The companywebsite example looks more like companyemail :)

**Comment by escopecz** [comment]
Hehe, you fixed the first issue before I submitted the comment, but the second still stands.

File: en/contacts/contact_monitoring.md [hunk 3]
```diff
@@ -62,6 +62,13 @@ You can attach custom parameters or overwrite the automatically generated parame
 ```
 This code will send all the automatic data to Mautic and adds also email and firstname. The values of those fields must be generated by your system. 
 
+From Mautic 2.13, tracking code support also company fields. Mautic assign company to your tracking contact based on company name. Then you have to add **company** or **companyname** parameter to tracking code along with other companies fields (companyemail, companyaddress1...):
```
[the remark this answers is not in the document]
Comment (dbhurley) [reply en/contacts/contact_monitoring.md]: In addition, you can also add other company parameters to the tracking code for other company fields.

**Pull request merged**